locales/en.yml

```diff
@@ -28,35 +30,32 @@ events.notgoing.btn: Not going
 events.venue: Venue Map
 events.source: Event Source
 events.views: Views
-events.register.title: Register
-events.register.submitlabel: Register
-events.login.btn: Login
 events.reservation.title: Almost there
 events.complete.btn: Complete registration
 events.reservation.description: See you soon! Don't forget to check-in by the organiser when you come!
 events.email.description: Check your email to finish creation of the WeDance profile.
 events.calendar.btn: Add to calendar
 
-events.edit.title: Add event
-events.edit.submitlabel: Save
+events.edit.submit.label: Save
 
 events.dashboard.empty: Event not found
 events.dashboard.description: Only event owner can access this area.
 events.dashboard.recipient: recipient | recipients
-events.dashboard.submitlabel: Send
-events.dashboard.composelabel: Compose email
-events.dashboard.participant-label: Add participant
-events.dashboard.email-label: Send emails
-events.dashboard.view-label: View Event
-events.dashboard.cancel-btn: Cancel
-events.dashboard.check-in: Check In
+events.dashboard.submit.label: Send
+events.dashboard.compose.label: Compose email
+events.dashboard.addGuest: Add Guest
+events.dashboard.participant.label: Add participant
+events.dashboard.email.label: Send emails
+events.dashboard.view.label: View Event
+events.dashboard.cancel.btn: Cancel
```

Comment:
`view.label` and `cancel.btn` are both buttons. pick one style please. i would just drop `btn` and `label`